src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-5-0-317.mdx

```diff
@@ -0,0 +1,27 @@
+---
+subject: PHP agent
+releaseDate: '2023-01-17'
+version: 10.5.0.317
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.5.0.317'
+---
+## New Relic PHP Agent v10.5.0.317
+
+### Bug Fixes
+* agent: Fixed [#577](https://github.com/newrelic/newrelic-php-agent/issues/577) - Large performance regression in 10.3.0. [#588](https://github.com/newrelic/newrelic-php-agent/pull/588)
+* agent: Fixed [#594](https://github.com/newrelic/newrelic-php-agent/issues/594) - Instrumentation from `opcache.preload` for PHP 8.1. [#595](https://github.com/new^relic/newrelic-php-agent/pull/595)
+* agent: Fixed sending `supportability` metrics for application logging configuration. [#575](https://github.com/newrelic/newrelic-php-agent/pull/575)
```

Comment:
The metrics names are `Supportability/Logging/Forwarding/PHP/{enabled|disabled}` and `Supportability/Logging/Metrics/PHP/{enabled|disabled}` so let’s either leave capital letter in, because these metrics belong to a wider group of metrics known as `Supportability` or let’s include full metrics names.

Reply:
@misiekn Is this your suggestion?

```suggestion
* agent: Fixed sending the following supportability metrics for application logging configuration [#575](https://github.com/newrelic/newrelic-php-agent/pull/575):
   * `Supportability/Logging/Forwarding/PHP/{enabled|disabled}`
   * `Supportability/Logging/Metrics/PHP/{enabled|disabled}`
```

Reply:
@a-sassman How about:
```suggestion
* agent: Fixed sending `Supportability/Logging/Forwarding/PHP/%` and `Supportability/Logging/Metrics/PHP/%` application logging configuration metrics. [#575](https://github.com/newrelic/newrelic-php-agent/pull/575)
```
Using `%` (wildcard) in the metric name makes it possible to be copied verbatim to be used in NRQL query to obtain the counts for both metrics (ending with enabled and disabled). I don't think `{enabled|disabled}` syntax is clear enough.

Reply:
@misiekn works for us!